src/interface/phpcs.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+var exec = require('child_process').exec;
+var _ = require('lodash');
+
+var phpcsInterface = function (options, f) {
+  var phpcs = exec(buildCommand(options), function (error, stdout) {
+    if (error) {
+      console.log(stdout);
+      f(new Error('Phpcs failed see errors above.'));
+    } else {
+      console.log('Everythings fine.');
+    }
+    f();
+  });
+
+  phpcs.stdin.end();
+};
+
+
+var buildCommand = function (opt) {
+  opt = opt || {};
+
+  var command = opt.bin || 'phpcs';
+  var commandOptionName;
+
+  var possibleOptions = [
+    'standard',
+    'warningSeverity',
+    'errorSeverity',
+    'encoding',
+    'standard'
+  ];
+
+  for (commandOptionName in possibleOptions) {
```

Comment:
il est recommandé d'utiliser for..in uniquement pour boucler sur les keys d'un object. 
Ici `commandOptionName` n'était pas le nom de la commande mais un index.

Pour des tableaux .forEach est plus adéquate :

```javascript
possibleOptions.forEach(function (commandOptionName) {
  if (_.has(opt, commandOptionName)) {
    command += proceedCommandDetection(opt, commandOptionName);
  }
});
```

Mais je préfèrerai même l'approche fonctionel, lazy (lodash v3) et donc sans side-effects :

```javascript
// curied helpers
var formatParameter = _.curry(function (opt, optionName) {
  return ' --' + _.kebabCase(optionName) + '="' + opt[optionName] + '"';
});

var has = _.curry(_.has, 2);

var parameters = _.chain(possibleOptions)
  .filter(has(opt))
  .map(formatParameter(opt))
  .value()
  .join('');

var fullCommand = command + parameters;
```